src/utils/schema_utils.ts

```diff
@@ -45,34 +55,38 @@ function schemaValidationErrorToValidationErrorItem(schemaValidationErrorObject:
             'dependencies',
         ].includes(schemaValidationErrorObject.keyword)
     ) {
-        return {
-            field: schemaValidationErrorObject.dataPath.replace('.', ''),
-            code: ValidationErrorCodes.IncorrectFormat,
-            reason: schemaValidationErrorObject.message || '',
-        };
+        field = schemaValidationErrorObject.dataPath.replace('.', '');
+        code =  ValidationErrorCodes.IncorrectFormat; 
     } else if (
         ['minimum', 'maximum', 'minLength', 'maxLength', 'minItems', 'maxItems', 'enum', 'const'].includes(
             schemaValidationErrorObject.keyword,
         )
     ) {
-        return {
-            field: schemaValidationErrorObject.dataPath.replace('.', ''),
-            code: ValidationErrorCodes.ValueOutOfRange,
-            reason: schemaValidationErrorObject.message || '',
-        };
+        field = schemaValidationErrorObject.dataPath.replace('.', '');
+        code = ValidationErrorCodes.ValueOutOfRange;
     } else if (schemaValidationErrorObject.keyword === 'required') {
-        return {
-            field: (schemaValidationErrorObject.params as AJV.RequiredParams).missingProperty,
-            code: ValidationErrorCodes.RequiredField,
-            reason: schemaValidationErrorObject.message || '',
-        };
+        field =  (schemaValidationErrorObject.params as AJV.RequiredParams).missingProperty;
+        code = ValidationErrorCodes.RequiredField;
     } else if (schemaValidationErrorObject.keyword === 'not') {
-        return {
-            field: schemaValidationErrorObject.dataPath.replace('.', ''),
-            code: ValidationErrorCodes.UnsupportedOption,
-            reason: schemaValidationErrorObject.message || '',
-        };
+        field =  schemaValidationErrorObject.dataPath.replace('.', '');
+        code = ValidationErrorCodes.UnsupportedOption;
     } else {
         throw new Error(`Unknown schema validation error name: ${schemaValidationErrorObject.keyword}`);
     }
+
+    if (schemaValidationErrorObject.description === undefined) {
+        return {
+            field: field,
+            code: code,
+            reason: reason,
```

Comment:
thought prettier might do this automatically for you, but can use this shorthand syntax: 
```suggestion
            field,
            code,
            reason,
```